pkg/infra/pulumi_aws/deploylib.ts

```diff
@@ -113,12 +113,8 @@ export class CloudCCLib {
         if (this.createVPC) {
             this.getVpcSgSubnets()
         }
-        const resolvedBucketName = this.account.accountId.apply(
-            (accountId) =>
-                sanitized(
-                    AwsSanitizer.S3.bucket.nameValidation()
-                )`${accountId}${physicalPayloadsBucketName}`
-        )
+        // Right now we sanitize this bucket name in the compiler go code so we do not need to here.
```

Comment:
```suggestion
        // Right now we sanitize this bucket name in the compiler go code so we do not need to here (issue #188)
```

Reply:
will add that but also the klotho-pro #51 is the true reason as to why we cant sanitize. We cant risk it being out of sync with the go code